elixirkit/demo/rel/appkit/Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version: 5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "Demo",
+    platforms: [
+        .macOS(.v11)
+    ],
+    dependencies: [
+        .package(name: "ElixirKit", path: "../../../elixirkit_swift")
```

Comment:
Is this the path to the dependency? If so, how would we make it work on an actual Elixir project where the dependency will have a version? One option would be to make this a `.eex` file and retrieve the path using `Mix.Project.deps_paths`?

Reply:
I though the path would then by like: ../../deps/elixirkit/elixirkit_swift and it would just work. Similiar to using phoenix.js i guess?

Reply:
It won't work for path dependencies but we can postpone until we cross that bridge.  :)